criterion.nix

```diff
@@ -1,9 +1,4 @@
-{ lib
-, fetchFromGitHub
-, cmake
-, pkg-config
-, criterion
-}:
+{ lib, fetchFromGitHub, cmake, pkg-config, criterion }:
```

Comment:
which formatter are you using? (not sure if the official one collapses back into a single line)

Reply:
nixfmt v0.6.0.
I just ran it across the repo and indeed everything was collapsed.

Do you use a different formatter? Could you let me know which one so I can format everything (or at least what I touched) with it

Reply:
think I used `nixfmt nixpkgs-unstable-2024-12-04`
You can also use the latest rfc https://discourse.nixos.org/t/nix-formatting-team-full-nixpkgs-reformat/61867

Reply:
Done with `nixfmt-tree`!